models/comments.models.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection");
+const { checkArticleExists } = require("./articles.models");
+
+exports.fetchCommentsByArticleId = (article_id) => {
+  const doesExist = checkArticleExists(article_id);
+  const query = db.query(
+    `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`,
+    [article_id]
+  );
+  return Promise.all([doesExist, query]).then((results) => {
+    if (results[1].rows.length === 0) {
+      return Promise.reject({
```

Comment:
I feel that sending back a 200 status inside a rejected Promise feels confusing. A 200 status code indicates that everything okay and nothing has gone wrong with the request. 

I understand your logic in that you want to be more specific in the responses that you are sending back to the client, but I think sending back a 200 status code and an empty array is perfectly satisfactory as it matches the behaviour of a successful request - which this would be.